src/CommandingOfficers/AWBW/OS/NellWarCrimes.java

```diff
@@ -0,0 +1,85 @@
+package CommandingOfficers.AWBW.OS;
+
+import java.util.ArrayList;
+
+import CommandingOfficers.*;
+import CommandingOfficers.CommanderAbility.CostBasis;
+import CommandingOfficers.AWBW.AWBWCommander;
+import Engine.GameScenario;
+import Engine.Combat.StrikeParams;
+import Engine.UnitMods.LuckBaseModifier;
+import Engine.UnitMods.UnitModifier;
+import UI.UIUtils;
+import Terrain.MapMaster;
+
+public class NellWarCrimes extends AWBWCommander
+{
+  private static final long serialVersionUID = 1L;
+
+  private static final CommanderInfo coInfo = new instantiator();
+  public static CommanderInfo getInfo()
+  {
+    return coInfo;
+  }
+  private static class instantiator extends CommanderInfo
+  {
+    private static final long serialVersionUID = 1L;
+    public instantiator()
+    {
+      super("Nell", UIUtils.SourceGames.AWBW, UIUtils.OS, "crime");
+      infoPages.add(new InfoPage(
+            "Nell (warcrimes)\n"
+          + "Nell, but she always rolls max luck.\n"
+          + "Luck on attacks is +19%.\n"));
```

Comment:
What kind of broken garbage is this? Dibs.

Reply:
It was requested for testing against such paragons of balance as Javier on a 2T map or Hachi in high funds.